### PR TITLE
upd(yabridge): `5.0.1` -> `5.0.3`

### PR DESCRIPTION
**Pull request by Zahrun**

### CODE REVIEW

**Comment by Henryws** [comment]
CI fails.

**Comment by Zahrun** [comment]
@echometerain

File: packages/yabridge/yabridge.pacscript [hunk 1]
```diff
@@ -1,9 +1,9 @@
 name="yabridge"
-version="5.0.1"
+version="5.0.3"
 url="https://github.com/robbert-vdh/yabridge/releases/download/${version}/yabridge-${version}.tar.gz"
 build_depends="libxcb1 libxcb1:i386"
```
Comment (echometerain) [inline packages/yabridge/yabridge.pacscript:4]: I don't think this required anymore
```suggestion
```

**Comment by echometerain** [comment]
Thanks for updating it, I should of done this yesterday tbh

**Review by Henryws — APPROVED** [review]

**Comment by Zahrun** [comment]
No problem, thanks for the script and the fix.
Actually, I discovered yabridge thanks to pacstall and you, and I’m very grateful for that. I was still using jbrigde which was much worse, so I’m grateful.

**Comment by Zahrun** [comment]
Also the new notification of outdated packages will help

**Pull request merged**